arrows/klv/klv_value.cxx

```diff
@@ -178,6 +378,51 @@ operator<<( std::ostream& os, klv_value const& rhs )
   return rhs.empty() ? os << "(empty)" : rhs.m_item->print( os );
 }
 
+// ---------------------------------------------------------------------------
+#define KLV_INSTANTIATE( T )                         \
```

Comment:
same comment as with `klv_data_format`.  You could put the template definitions and this macro in a `.hxx` file and then include it in several `.cxx` files to instantiate different types so this `klv_value` code doesn't have to be entangled with all the different types you want to instantiate.